weblogic-azure-aks/src/main/arm/scripts/appgw-helm-config.yaml.template

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2021, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# Based on https://raw.githubusercontent.com/Azure/application-gateway-kubernetes-ingress/master/docs/examples/sample-helm-config.yaml
+
+# This file contains the essential configs for the ingress controller helm chart
```

Comment:
helm -> Helm

Capitalize proper nouns (in this case, a product name)

Reply:
Fix all product name of Helm in https://github.com/galiacheng/weblogic-azure/commit/6d2890a28167f5108a1d3d85dcaf6661faf009db